external/fv3fit/tests/emulation/test_losses.py

```diff
@@ -5,20 +5,24 @@
     CustomLoss,
     NormalizedMSE,
 )
+from fv3fit.emulation.layers.normalization2 import (
+    NormFactory,
+    ScaleMethod,
+    CenterMethod,
+    NormLayer,
+)
 
 
 def test_NormalizeMSE():
-    sample = np.array([[25.0], [75.0]])
-    target = np.array([[50.0], [50.0]])
-
-    mse_func = NormalizedMSE("mean_std", sample)
-    mse = mse_func(target, sample)
-    np.testing.assert_approx_equal(mse, 1.0, 6)
+    sample = np.array([[0], [0]])
```

Comment:
`NormalizedMSE` is simple enough now that I don't think it's worth checking the actual values.